### DISCUSSION TITLE
Explained Variance

### DISCUSSION THREAD

**Discussion by rootonchair** [mlbvn/handson-ml2-vn#81] [Terminology]
Em đang revise phần chương 8 thì có khái niệm "explained variance ratio".
Theo định nghĩa của "explained variance":
> In statistics, explained variation measures the proportion to which a mathematical model accounts for the variation (dispersion) of a given data set

Hiện từ này đang được dịch là "phương sai được giải thích". Em thấy từ này chưa có trong glossary nên nếu mọi người thấy ok thì mình thêm vô ạ.

cc: @mlbvn/handson-ml2

**Comment by cuongvng** [comment]
có tính từ nào ổn không? Mình thấy dịch dạng động từ bị động sẽ nhập nhằng những cases như là:

`phương sai được giải thích` `là đại lượng mô tả ...` có thể đọc sang `phương sai` `được giải thích là đại lượng mô tả...`
